### PR TITLE
Show geostyler version in footer

### PR DESCRIPTION
**Pull request by jansule**
Title says it all...

image

### CODE REVIEW

**Review by hwbllmnn — APPROVED** [review]

**Review by KaiVolland — APPROVED** [review]
nice

File: public/index.html [hunk 1]
```diff
@@ -49,10 +49,14 @@
...
           Imprint
         </a>
-        –
+        —
         <a href="https://www.terrestris.de/en/datenschutzerklaerung">
           Privacy Policy
         </a>
+        —
+        <a href="https://github.com/geostyler/geostyler/releases/tag/v<%= htmlWebpackPlugin.options.geostylerVersion %>">
+          GeoStyler <%= htmlWebpackPlugin.options.geostylerVersion %>
```
Comment (marcjansen) [inline public/index.html:58]:
```suggestion
          GeoStyler v<%= htmlWebpackPlugin.options.geostylerVersion %>
```

Maybe we also want to link to the release page?: https://github.com/geostyler/geostyler/releases/tag/v10.1.0

**Review by marcjansen — APPROVED** [review]

**Pull request merged**